rules-tests/DowngradePhp70/Rector/Class_/DowngradeAnonymousClassRector/Fixture/with_args.php.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Rector\Tests\DowngradePhp70\Rector\Class_\DowngradeAnonymousClassRector\Fixture;
+
+class WithArgs
+{
+    public function run($arg1, $arg2)
+    {
+        return new class ($arg1, $arg2) {
```

Comment:
@TomasVotruba anonymous class with arguments seems not updateable with : 

```php
new Name(sprintf('Anonymous(%s)', $argsString))
```

Do you have idea how to replace parent node (always a `New_` instance) like the following:

```php
        $parent = $class->getAttribute(AttributeKey::PARENT_NODE);
        $parent = new New_(new Name('Anonymous'), $args);
```

Reply:
@TomasVotruba finally works with `New_` for node type 🎉